clang/lib/Headers/hlsl/hlsl_intrinsics.h

```diff
@@ -916,7 +916,7 @@ float4 lerp(float4, float4, float4);
 /// \brief Returns the length of the specified floating-point vector.
 /// \param x [in] The vector of floats, or a scalar float.
 ///
-/// Length is based on the following formula: sqrt(x[0]^2 + x[1]^2 + �).
+/// Length is based on the following formula: sqrt(x[0]^2 + x[1]^2 + �).
```

Comment:
This looks unintended?

Reply:
I keeps coming back whenever I save the file. Seems like VSCode cannot render this character and replaces it with ?. Github web also cannot render this [character](https://github.com/llvm/llvm-project/blob/f71b63865140cf3c286baf3a77ba3e467f929504/clang/lib/Headers/hlsl/hlsl_intrinsics.h#L919)  it looks just like whitespace.

Reply:
The character was … originally introduced here https://github.com/llvm/llvm-project/pull/101256. But it somehow got managled to a non-printable character.

Reply:
addressed in https://github.com/llvm/llvm-project/commit/6a38e19c92ed09eeecb70d5f61c3b822acb4964d